tools.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import "github.com/joho/godotenv"
+
+// Load current .env file into map
+func loadCurEnv() (EnvDataMap, error) {
+	var env EnvDataMap
+	env, err := godotenv.Read()
+	if err != nil {
+		return nil, err
+	}
+	return env, nil
```

Comment:
as written in #2 I would keep this in the execute function as this function is almost just a wrapper. in addition its missing the file not found check